docs/content/docs/format/api/auto.um

```diff
@@ -134,3 +164,166 @@
         @description
           The formatted string
 
+@function hx.round
+  @added nextReleaseVersion
+    @issue 457
+    @description
+      Added plain functions for formatting numbers that take the number and
+      return the formatted number (as opposed to a curried function)
+
+  @description
+    A function to round a number to a specific number of significant figures.
+    @@codeblock js
+      hx.format.round(4)(3.1415926) // Returns "3.142"
```

Comment:
Needs updating to match the new api

Reply:
Same for the other new functions as well

Reply:
Done